### PR TITLE
docs(cli): new command debug

### PR DESCRIPTION
**Pull request by chobits**
### Description

Docs for: https://github.com/Kong/kong-ee/pull/6485

 

### Testing instructions

Preview link: 
https://github.com/Kong/docs.konghq.com/blob/cli-kong-debug/app/_src/gateway/reference/cli.md#kong-debug

https://deploy-preview-6344--kongdocs.netlify.app/gateway/latest/reference/cli/#kong-debug

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

[KAG-2375](https://konghq.atlassian.net/browse/KAG-2375)

[KAG-2375]: https://konghq.atlassian.net/browse/KAG-2375?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Hayk-S** [comment]
Please hold merging this PR until the resolution of https://konghq.atlassian.net/browse/KAG-2840 as some changes may be required.

File: app/_src/gateway/reference/cli.md [hunk 1]
```diff
@@ -378,4 +378,85 @@ Options:
...
+       the command output.
+
+ `2` - In progress. The profiling is still in progress.
+       The following commands make use of this return value:
+       - kong debug profiling cpu start
+       - kong debug profiling memory start
+       - kong debug profiling gc-snapshot
+```
+
+---
```
[the remark this answers is not in the document]
Comment (chobits) [reply app/_src/gateway/reference/cli.md:459]: TY, I will modify it to make the message more friendly

[the remark this answers is not in the document]
Comment (chobits) [reply app/_src/gateway/reference/cli.md:459]: FYI, why we cannot change log level when not using a database, we can see this comment: https://github.com/Kong/kong/pull/9744/files#r1025226826

[the remark this answers is not in the document]
Comment (chobits) [reply app/_src/gateway/reference/cli.md:474]: resolved in https://github.com/Kong/docs.konghq.com/pull/6344/commits/84f3ad63473266c687062d03767ab1bfdadafd79

**Comment by chobits** [comment]
> ⚠️ This PR edits generated files. Please make sure that the source file is updated.
> 
> **https://github.com/Kong/kong/tree/master/kong/cmd:**
> 
> * app/_src/gateway/reference/cli.md

hi @cloudjumpercat 
I noticed this file `cli.md` has been generated by the kong CE `https://github.com/Kong/kong/tree/master/kong/cmd:**`.

The new kong debug CLI tool is not available in the CE version; it's exclusively an EE feature. Therefore, I manually added the `kong debug` usage section to the `cli.md`. Is that acceptable?

**Comment by cloudjumpercat** [comment]
> > ⚠️ This PR edits generated files. Please make sure that the source file is updated.
> > **https://github.com/Kong/kong/tree/master/kong/cmd:**
> > 
> > * app/_src/gateway/reference/cli.md
> 
> hi @cloudjumpercat
> 
> I noticed this file `cli.md` is generated by the kong CE `https://github.com/Kong/kong/tree/master/kong/cmd:**`. The new kong debug CLI tool is not available in the CE version; it's exclusively an EE feature. Therefore, I manually added the `kong debug` usage section to the `cli.md`. Is that acceptable?

Hi @chobits ! I think the manual edits might be erased since the file is autogenerated. Could you try adding it to https://github.com/Kong/kong-ee/tree/master/kong/cmd and we'll generate from the EE docs instead?

**Comment by chobits** [comment]
> > > ⚠️ This PR edits generated files. Please make sure that the source file is updated.
> > > **https://github.com/Kong/kong/tree/master/kong/cmd:**
> > > 
> > > * app/_src/gateway/reference/cli.md
> > 
> > 
> > hi @cloudjumpercat
> > I noticed this file `cli.md` is generated by the kong CE `https://github.com/Kong/kong/tree/master/kong/cmd:**`. The new kong debug CLI tool is not available in the CE version; it's exclusively an EE feature. Therefore, I manually added the `kong debug` usage section to the `cli.md`. Is that acceptable?
> 
> Hi @chobits ! I think the manual edits might be erased since the file is autogenerated. Could you try adding it to https://github.com/Kong/kong-ee/tree/master/kong/cmd and we'll generate from the EE docs instead?

Yes. It has been involved in kong-ee/kong/cmd directory. So its unnecessary for this PR, we can just close it.

**Comment by cloudjumpercat** [comment]
> BTW, I found one comment from cli.md which says that it is generated from CE kong/cmd ([source link](https://github.com/Kong/docs.konghq.com/blob/main/app/_src/gateway/reference/cli.md?plain=1#L5C1-L5C1)). We need to ensure that cli.md will be generated by EE kong/cmd.

@chobits We decided to put the CE link in that file because the general public don't have access to EE repos. The docs team will be talking about a better solution for this. But we'll make sure your changes are added when we autogenerate the CLI file in our release PR, so I'll close this PR for now.

**Pull request closed without merging**